### PR TITLE
Station Sum enhancements

### PR DESCRIPTION
**Pull request by dsteinich**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -77,6 +85,12 @@
             <version>1.3.0</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```
Comment (mbucknell) [inline pom.xml]: Should we be adding this to the other etl projects?

File: src/test/java/gov/acwi/wqp/etl/BaseFlowIT.java [hunk 2]
```diff
@@ -75,6 +75,7 @@ public void baseSetup() {
 				.addString("wqpDataSourceId", "1", true)
 				.addString("wqpDataSource", "stewards", true)
```
Comment (mbucknell) [inline src/test/java/gov/acwi/wqp/etl/BaseFlowIT.java]: Are the specific mentions of stewards going to disappear form wqp-etl-core?

Comment (dsteinich) [reply src/test/java/gov/acwi/wqp/etl/BaseFlowIT.java]: WQP-1397

File: pom.xml (hunk 1 above)
Comment (dsteinich) [reply pom.xml]: Possibly? I saw a message somewhere recommending it - either in eclipse or as part of a test run's console output.

**Pull request merged**